pkg/ddc/jindofsx/transform.go

```diff
@@ -188,7 +190,7 @@ func (e *JindoFSxEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *J
 	return value, err
 }
 
-func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, metaPath string, value *Jindo, dataset *datav1alpha1.Dataset, smartdataTag string, fuseTag string) (err error) {
+func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, metaPath string, value *Jindo, dataset *datav1alpha1.Dataset, newVersionSupport bool) (err error) {
```

Comment:
How about rename newVersionSupport to secretMountSupport?

Reply:
DONE